packages/third-parties/agenda/src/AgendaModule.spec.ts

```
@@ -137,6 +137,46 @@
         expect(agendaModule.agenda.close).toHaveBeenCalledWith({force: true});
       });
     });
+
+    describe("_collection", () => {
+      it("should expose _collection", () => {
+        const agendaModule = PlatformTest.get<any>(AgendaModule)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)